src/pages/objkt-display/index.js

```diff
@@ -125,6 +127,36 @@ export const ObjktDisplay = () => {
   }, [])
 
   const Tab = TABS[tabIndex].component
+
+  const objkt_classes = []
```

Comment:
i think this needs to be put in `useState`.

Reply:
Isn't making the useEffect depend on nft alleviates that?

Reply:
I've done all besides this as I'm not sure ;)

Reply:
hmm, well, if it works, it works :)

as soon as you add something async to the effect, it will probably break though.